daemon/errors.go

```diff
@@ -64,6 +64,20 @@ func errExecPaused(id string) error {
 	return stateConflictError{cause}
 }
 
+type conflictError struct {
```

Comment:
Maybe we can use a dedicated error for name conflict instead of introducing a new generic conflict?